tools/plugin-base/src/main/java/io/spine/tools/gradle/ProtocConfigurationPlugin.java

```diff
@@ -103,32 +99,13 @@ private void configureProtocTasks(GenerateProtoTaskCollection tasks) {
      *
      * @param plugins
      *         container of all plugins
+     * @param project
```

Comment:
Please have a description. In particular, in the context of the current issue, it may be either a root project or a project of a child Gradle module.